src/sentry/eventstream/kafka/backend.py

```diff
@@ -136,7 +136,10 @@ def insert(self, group, event, is_new, is_sample, is_regression,
             'project_id': event.project_id,
             # TODO(mitsuhiko): We do not want to send this incorrect
             # message but this is what snuba needs at the moment.
+            'title': event.title,
```

Comment:
If these properties exist in the event body (`event.data.items()`) lets not duplicate them here.

Reply:
@alex-hofsteede they are not in event.data.items, but they are in event.as_dict. Ideally we can consistently use the latter.